synapse/handlers/sso.py

```diff
@@ -94,3 +99,98 @@ async def get_sso_user_by_remote_user_id(
 
         # No match.
         return None
+
+    async def get_mxid_from_sso(
+        self,
+        auth_provider_id: str,
+        remote_user_id: str,
+        user_agent: str,
+        ip_address: str,
+        mapper: "Callable[..., Awaitable[str]]",
+        *args: Any,
+        **kwargs: Any,
+    ):
+        """
+
```

Comment:
But what does it dooooooooooooooooooooooooooooooooo?

Reply:
Hopefully the above makes sense!